example/ios/KlaviyoReactNativeSdkExample/AppDelegate.mm

```diff
@@ -10,38 +10,38 @@ @implementation AppDelegate
 BOOL isDebug = YES;
 
 // Change to NO if you prefer to initialize and handle push tokens in the React Native layer
-BOOL useNativeImplementation = YES;
```

Comment:
Intentional? If yes probably should update the comment right above.

Reply:
definitely not intentional, i think i have a couple test things in this PR that need to switched back!